src/neural/onnx/converter.cc

```diff
@@ -159,6 +160,14 @@ class Converter {
       const std::vector<float>&, std::initializer_list<int> dims,
       std::initializer_list<int> order = {});
 
+  std::unique_ptr<OnnxConst> GetScalarConverter(float in);
+
+  std::string OptionalBf16FixPrologue(OnnxBuilder* builder, std::string flow,
+                                      std::string name);
+
+  std::string OptionalBf16FixEpilogue(OnnxBuilder* builder, std::string flow,
```

Comment:
Two functions have long but too similar names (only three letters in the middle are different). Was a bit confusing to read for me, but fine to keep as it is.

Reply:
Renamed to `StartOptionalBf16Fix` and `EndOptionalBf16Fix`.